test_util/azure_test_driver.py

```diff
@@ -124,14 +120,17 @@ def main():
         print("Creating template deployment ...")
         deploy_poller = rmc.deployments.create_or_update(group_name, deployment_name, deployment_properties)
 
-        # Stop after 45 attempts (each one takes up to one minute)
-        @retry(stop_max_attempt_number=45)
+        def retry_if_assertion_error(exception):
+            """Return True if we should retry (in this case when it's an AssertionError), False otherwise"""
+            return isinstance(exception, AssertionError)
+
+        @retry(retry_on_exception=retry_if_assertion_error, stop_max_attempt_number=45)
```

Comment:
How frequently will the retry be attempted? I see it's limited to 45 attempts, but not sure on the retry interval.